src/global.ts

```diff
@@ -159,8 +159,8 @@ export interface Auth0ClientOptions extends BaseLoginOptions {
   useRefreshTokens?: boolean;
 
   /**
-   * If true, fallback to the technique of using a hidden iframe and the `authorization_code` grant with `prompt=none` when unable to use refresh tokens.
-   * The default setting is `true`.
+   * If true, fallback to the technique of using a hidden iframe and the `authorization_code` grant with `prompt=none` when unable to use refresh tokens. If false, the iframe fallback is not used and
+   * errors relating to a failed Refresh Grant should be handled appropriately by having the user to login. The default setting is `false`.
```

Comment:
As we talk about `authorization_code` grant, I would also mention `refresh_token` grant, instead of Refresh Grant.

I would also leave out the `by having the user to login.`, as some might want to redirect to a page or anything else when these errors happen.

Reply:
Changed `Refresh Grant` here and below to `refresh_token` and removed that pieces of text